charts/nri-kube-events/Chart.yaml

```diff
@@ -8,8 +8,8 @@ sources:
   - https://github.com/newrelic/nri-kube-events/tree/main/charts/nri-kube-events
   - https://github.com/newrelic/infrastructure-agent/
 
-version: 2.2.10
-appVersion: 1.8.2
+version: 2.2.11
```

Comment:
should we bump the minor if the minor of the app was bumped?

Reply:
It makes sense, I'm updating it. Thanks!

Reply:
I would not revert the change but as the versions are not aligned, the feature version of the chart should add a feature **in the chart**, not the application below.

But this is a long conversation to have when we start to add the release toolkit to these repositories...

Reply:
I think the criteria can be confusing when we release dependencies updates and, in the case of helm-charts, it is even more confusing because the application can be seen as one of the dependencies... As you said, we can unify (and actually implement) that criteria when we add the release toolkit. I'm looking forward to having that conversation 😄